frontend/app-development/layout/Layout.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Outlet, matchPath, useLocation } from 'react-router-dom';
+import { PageHeader } from './PageHeader';
+
+/**
+ * Displays the layout for the app development pages
+ */
+export const Layout = (): React.ReactNode => {
```

Comment:
Not sure if I think "layout" is the best name for this component. All it does is to render `<PageHeader/>` and the child routes. What about "AppShell"? 🤔